roles/cinder-data/tasks/main.yml

```diff
@@ -24,17 +24,23 @@
 - name: ensure tgt service is running
   service: name=tgt state=started
 
-# TODO: allow for device-backed VGs in addition to file-backed VGs
-- name: cinder volume group
-  cinder_volume_group: file={{ cinder.volume_file }}
-                       size={{ cinder.volume_file_size }}
+- include: volume_group.yml
   when: cinder.fixed_key is not defined or (cinder.fixed_key is defined and "compute" not in group_names)
-  # a fixed_key cannot be used with a local cinder volume group on a compute
-  # node. This would be a security risk, DO NOT CHANGE.
 
 - name: iscsi start
   service: name=open-iscsi state=started
 
+- name: enable cinder encryption
+  template: src=etc/cinder/cinder.encryption.conf dest=/etc/cinder/cinder.encryption.conf mode=0640
```

Comment:
Why is this a separate file? Is there a precedent for using periods as word separators in OpenStack configuration files?